aws/resource_aws_cloudwatch_event_target_test.go

```diff
@@ -1107,14 +1107,31 @@ resource "aws_cloudwatch_event_target" "test" {
   input_transformer {
     input_paths = {
       time = "$.time"
+      severity : "$.detail.severity",
+      Finding_ID : "$.detail.id",
+      instanceId : "$.detail.resource.instanceDetails.instanceId",
+      port : "$.detail.service.action.networkConnectionAction.localPortDetails.port",
+      eventFirstSeen : "$.detail.service.eventFirstSeen",
+      eventLastSeen : "$.detail.service.eventLastSeen",
+      count : "$.detail.service.count",
+      Finding_Type : "$.detail.type",
+      region : "$.region",
     }
```

Comment:
It would be good here to have one test that passes, plus a test that has more than 10 paths to demonstrate that it fails, using `ExpectError` in the test step

Reply:
good shout! Have added this in the latest commit